packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -319,20 +318,27 @@ export function DataviewsViewConfigDropdown() {
 					paddingSize="medium"
 					className="dataviews-config__popover-content-wrapper"
 				>
-					<VStack className="dataviews-view-config" spacing={ 6 }>
+					<Stack
+						direction="column"
+						className="dataviews-view-config"
+						gap="lg"
+					>
 						<SettingsSection title={ __( 'Appearance' ) }>
-							<HStack expanded className="is-divided-in-two">
+							<Stack
```

Comment:
I'm just checking the code first and will test the UI, but I'm curious about the `expanded` prop migration. Before it was set as `default` and added some width/height styles:
```
height: isColumn && expanded ? '100%' : undefined,
width: ! isColumn && expanded ? '100%' : undefined,
```

So, at first reading I'd expect some additions to every usage that used the default or like here set it explicitly to `true`.